Phonebook.Frontend/src/app/modules/table/components/table/table.component.ts

```diff
@@ -48,10 +53,22 @@ export class TableComponent implements OnInit, OnDestroy {
     private router: Router,
     private personService: PersonService,
     public dialog: MatDialog,
-    public store: Store
+    public store: Store,
+    public breakpointObserver: BreakpointObserver
   ) {}
 
   public ngOnInit() {
+    this.breakpointObserver
+      .observe('(max-width: 900px)')
+      .pipe(untilComponentDestroyed(this))
+      .subscribe((result) => {
+        this.isSizeSmall = result.matches;
+        if (this.isSizeSmall) {
```

Comment:
I explicitly mean the return statement. The code should work if you leave out return in the following statement. ( return this.isSizeSmall === true;)

But also you are setting `this.isSizeSmall` before and based on the setting your are setting it again...